sdk/cosmosdb/cosmos/api-extractor.json

```diff
@@ -1,24 +1,31 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/api-extractor.schema.json",
-  "compiler": {
-    "configType": "tsconfig",
-    "rootFolder": "./src"
-  },
-  "project": {
-    "entryPointSourceFile": "../dist-esm/index.d.ts"
+  "mainEntryPointFilePath": "./dist-esm/index.d.ts",
+  "docModel": {
+    "enabled": false
```

Comment:
Updated `"@microsoft/api-extractor"` to version `"^7.1.5"` to match with the other packages and also had to update the api-extractor.json to get it working.